app/views/notification_mailer/report_failure_email.text.erb

```diff
@@ -4,7 +4,6 @@ This task has either reached the maximum number of attempts, or is not suitable
 
 Vendor: <%=@integration.vendor_name %>
 Batch ID: <%=@task.batch_id %>
-Completion Type: <%=@action %>
```

Comment:
![](http://www.reactiongifs.com/r/but-why.gif)

Reply:
I realised the `task` object (`Scheduler`) doesn't have knowledge of the action. The error message below will include detail on the reason for failure.

I can plumb that in if you think it's necessary.

Reply:
Yes, let's add it please.